src/projectscene/view/clipsview/trackslistclipsmodel.cpp

```diff
@@ -30,15 +30,20 @@ void TracksListClipsModel::load()
 
     m_trackList = prj->trackList();
 
-    m_dataSelectedTracks = selectionController()->dataSelectedOnTracks();
-    m_selectedTracks = selectionController()->selectedTracks();
+    selectionController()->tracksSelected().onReceive(this, [this](const trackedit::TrackIdList& tracksIds) {
+        Q_UNUSED(tracksIds);
+        emit dataChanged(index(0), index(m_trackList.size() - 1), { IsTrackSelectedRole });
```

Comment:
If `m_strackList.size() == 0`, `index(m_trackList.size() - 1)` will wrap around to an enormous number, won't it? It that's intended, please add a comment, otherwise typecast to index before subtracting.